task5.css

```diff
@@ -0,0 +1,80 @@
+body {
```

Comment:
font looks not good 
![image](https://user-images.githubusercontent.com/40792121/78402552-cd577480-760b-11ea-9fbd-f7a8dd2cae98.png)